### ISSUE TITLE
Push documentation upstream

### ISSUE THREAD

**Issue by phoe** [cl-library-docs/common-lisp-libraries#2]
Some parent projects, such as [Bordeaux Threads](https://github.com/sionescu/bordeaux-threads/issues/53), could use documentation updates in their main repositories when it is complete.

**Comment by digikar99** [comment]
For the most part, the documentation is taken from the docstrings. (alexandria and postmodern are an exception atm.) However, there does go much manual efforts in hyperlinking or formatting, due to either inconsistent formatting, or wanting to avoid hyperlink-cognitive-overload for the reader. 

And let me clarify the title of this issue: you want the official documentation to be used here - is that the case?

**Comment by phoe** [comment]
Nope, I want the inverse - the official documentation is often out of date and could use efforts that update it, with BT being one example that I am aware of.
